classes/instructions.php

```diff
@@ -0,0 +1,79 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * This file contains the page view if the user has the capability 'perform_observations'
+ *
+ * @package   mod_observation
+ * @copyright  2021 Endurer Solutions Team
+ * @author Matthew Hilton <mj.hilton@outlook.com>
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace mod_observation;
+
+defined('MOODLE_INTERNAL') || die();
+
+/**
+ * mod_observation functions for processing instructions
+ *
+ * @package   mod_observation
+ * @copyright  2021 Endurer Solutions Team
+ * @author Matthew Hilton <mj.hilton@outlook.com>
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class instructions {
+    /**
+     * Generates a formatting block of HTML to output to a page that displays the instructions passed
+     *
+     * @param string|null $heading Instructions heading
+     * @param string|null $bodytext Instructions body text
+     * @param int|null $bodyformat Format for the body (moodle format identifier)
+     * @param int $headinglevel HTML heading level
+     * @param string|null $defaultmessage Message to display if there were no instructions given.
+     * Defaults to lang string ['defaultmessagenoinstructions]
+     * @return string formatted html to be displays encoded as a string
+     **/
+    public static function observation_instructions(
+        $heading,
```

Comment:
Can we change the declaration style a bit: https://docs.moodle.org/dev/Coding_style#Wrapping_lines
Basically fill up lines as much as practical, with extra indentation